Dockerfile-alpine.template

```
@@ -119,13 +107,17 @@ RUN set -eux; \
 # Compile, install OpenSSL, verify that the command-line works & development headers are present
 	make -j "$(getconf _NPROCESSORS_ONLN)"; \
 	make install_sw install_ssldirs; \
-	cd ..; \
-	rm -rf "$OPENSSL_PATH"*; \
 # use Alpine's CA certificates
 	rmdir "$OPENSSL_CONFIG_DIR/certs" "$OPENSSL_CONFIG_DIR/private"; \
 	ln -sf /etc/ssl/certs /etc/ssl/private "$OPENSSL_CONFIG_DIR"; \
 # smoke test
-	openssl version; \
+	openssl version
```

Since we're multi-stage, it doesn't hurt us to add an extra layer here, and what we get in return is the ability to inspect the failed compilation a little:

```suggestion
	ln -sf /etc/ssl/certs /etc/ssl/private "$OPENSSL_CONFIG_DIR"

￼# smoke test
RUN openssl version
```

---

Great idea